### ISSUE TITLE
use of terraform native resources for GWLB terraform deployment

### ISSUE THREAD

**Issue by tbugfinder** [PaloAltoNetworks/AWS-GWLB-VMSeries#13]
Request:

I'm wondering if this repo could use native terraform provider AWS resources instead of just relying on an scripted API deployment.

REF: https://github.com/hashicorp/terraform-provider-aws/issues/16129 

https://github.com/PaloAltoNetworks/AWS-GWLB-VMSeries/blob/9194f4871699c5075c193a9df64334f46ddd1cc5/terraform/security_stack/gwlb.tf#L35

**Comment by georgegil** [comment]
indeed... tbh this repo didn't really impress me, so I redid the whole thing in native terraform 

https://github.com/georgegil/terraform-aws-paloalto-firewall-asg

ok and swapped all the lamda python for poweshell which reads a tad easier...
